tests/functional/spec/interstitial_definition.spec.js

```diff
@@ -8,36 +8,20 @@ describe("Component: Interstitial Definition", () => {
 
     it("When there is a definition on an interstitial, then the page is displayed correctly", () => {
       expect($(InterstitialDefinitionPage.definitionTitle(1)).isDisplayed()).to.be.true;
-      expect($(InterstitialDefinitionPage.definitionContent(1)).isDisplayed()).to.be.false;
-      expect($(InterstitialDefinitionPage.definitionButton(1)).isDisplayed()).to.be.false;
+      expect($(InterstitialDefinitionPage.definitionContent(1)).getText()).to.equal("");
```

Comment:
Same here, this will always pass.

Reply:
The previous one did need changing because it was using `contain` but as this one is using `equal` it is working correctly already